apps/myaccount/src/configs/app.ts

```diff
@@ -18,7 +18,7 @@
 
 import { I18nModuleOptionsInterface } from "@wso2is/i18n";
 import { I18nConstants } from "../constants";
-import { SCIMConfigs } from "../extensions/configs";
+import { SCIMConfigs } from "../extensions/configs/scim"; //keep statement as this in order to avoid cyclic dependency
```

Comment:
```suggestion
// Keep statement as this to avoid cyclic dependency. Do not import from config index.
import { SCIMConfigs } from "../extensions/configs/scim";
```